docs/DevEnv.md

```diff
@@ -17,9 +17,40 @@ Otherwise, check or change its location in the preferences in the Arduino IDE.
 The Arduino IDE lacks the major features of a professional IDE.
 Instead, you're encouraged to work with VSCode and the appropriate extension.
 
+
 You'll find the extension by typing `Arduino` in the VSCode search extension field 
 (`Ctrl+Shift+X`) and clicking the one provided by Microsoft.
 
+Next, copy the `libraries` directory from the repo to `~/Arduino/libraries/` like so:
+
+```bash
+cp -r libraries ~/Arduino/libraries
+```
+
+You will also need to update the `includePath` by adding the path the `c_cpp_properties.json` file:
```

Comment:
You might* also need [...]
On Linux, even with a fresh installation without the Arduino IDE, I didn't need to do that.

Plus, you didn't say where to put this file. Is this a VSCode config ?